script/smt-report

```diff
@@ -495,64 +432,22 @@ $LOG = SMT::Utils::openLog($logfile);
 # calculate best report
 my $cfg = SMT::Utils::getSMTConfig();
 
-if ( $cfg->val('NU', 'ApiType', 'NCC') eq 'NCC')
+my $api = $cfg->val('NU', 'ApiType', 'NCC');
+if ( $api eq 'NCC')
 {
-    printLog($LOG, $vblevel, LOG_ERROR, __("NCC is not supported in this version of SMT"));
+    printLog($LOG, $vblevel, LOG_ERROR, __("NCC is not supported by this version of SMT"));
     exit 1;
 }
-
-# sync current data from SCC
-if(!$nosccsync)
+if ( $api eq 'SCC' && $nccreport)
```

Comment:
Compilation fail on undeclared $nccreport.
I noticed --scc option was removed from GetOptions and $sccreport went away with that. Should I removed this whole section or put back --scc only to be able to print this warning? Usage string needs updating too.